network/tls.go

```diff
@@ -184,6 +185,7 @@ func (cm *certMaker) get(nonce []byte) (*tls.Certificate, error) {
 	// test code to get ahold of and modify.
 	if testNoURIs {
 		tmpl.URIs = nil
+		tmpl.MaxPathLen = 1
```

Comment:
This is necessary as a true repro of v3.2.4 behaviour, but it is not important. It will cause Go 1.15's server-side TLS to fail with an "internal error".